client/disputes/new-evidence/index.tsx

```diff
@@ -786,6 +787,8 @@ export default ( { query }: { query: { id: string } } ) => {
 							onClick={ () =>
 								handleStepChange( currentStep + 1 )
 							}
+							icon={ <Icon icon={ chevronRight } /> }
```

Comment:
Nitpick: it seems we can also use like this without an additional `Icon` component. 

```suggestion
							icon={ chevronRight }
```

Reply:
that's true, thanks 1cf49acf0c0a731b80af0d02068e87eba0fd04ce